app/helpers/audits_helper.rb

```diff
@@ -13,7 +13,7 @@ def id_to_label(name, change, truncate = true)
       when /.*_id$/
         label = name.classify.gsub('Id','').constantize.find(change).to_label
       else
-        label = change.to_s == "[encrypted]" ? _(change.to_s) : change.to_s
+        label = change.to_s == "[redacted]" ? _(change.to_s) : change.to_s
```

Comment:
should be using `AuditExtensions::REDACTED`?

Reply:
fixed